### ISSUE TITLE
Enable certificate error assistant

### ISSUE THREAD

**Issue by jumde** [brave/brave-browser#13010]
## Description
Certificate error assistant is currently not installed in brave. This component presents special interstitial pages for pages like mitm-software or captive-portal.

**Comment by mherrmann** [comment]
Similar to #12999: At the moment, Certificate Error Assistant appears in brave://components as version 0.0.0.0 and stays this way forever. My fix for #8709 will actually remove this zombie appearance. But whoever implements this present issue to bring it back can simply look at my fix and remove Certificate Error Assistant (ID `giekcmmlnklenlaomppkphknjmnnpneh`) from the list of blacklisted components.

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.20.87 Chromium: 88.0.4324.96 (Official Build) dev (x86_64)
-- | --
Revision | 68dba2d8a0b149a1d3afac56fa74648032bcf46b-refs/branch-heads/4324@{#1784}
OS | macOS Version 10.15.7 (Build 19H15)
```

Verified as part of https://github.com/brave/brave-browser/issues/12999#issuecomment-765699323

<img width="290" alt="cert err ast - installed" src="https://user-images.githubusercontent.com/28145373/105551116-54c3c080-5cd0-11eb-92cb-0dcb3eb44037.png">

----------------
